docs/sphinx/manual/building_hpx.rst

```diff
@@ -23,11 +23,11 @@ This section gives an introduction on how to use our build system to build
 CMake basics
 ============
 
-|cmake|_ is a cross-platform build-generator tool. cmake does not build the
+|cmake|_ is a cross-platform build-generator tool. CMake does not build the
```

Comment:
Do you think it's useful to have a link every time we mention e.g. CMake? I've generally tried to make terms like that links *every* time they're mentioned but I haven't been very successful it seems.